kivy/core/text/markup.py

```diff
@@ -752,7 +752,14 @@ def p_restricted(line, uw, c):
         s = self.get_extents(last_text)
         if len(last_text):
             line1.append(LayoutWord(last_word.options, s[0], s[1], last_text))
-        elps.options = last_word.options
+
+        elps_opts = old_opts
```

Comment:
I can't see how `elps_opts` will affect the `textwidht()` function. Nonetheless, I made the change but it didn't fix the variable font case.